src/style.css

```diff
@@ -29,3 +32,18 @@ h1 {
 .hello {
   display: none;
 }
+
+@media screen and (max-width: 800px) {
+  body {
+    font-size: 16px;
+  }
+
+  .container {
+    font-size: 8px;
+    margin: 20px;
+  }
+
+  input {
+    width: 70%;
+  }
+}
```

Comment:
The styling of your application could be improved a lot. Right now, the layout is bland and barely matches the wireframe provided by the Microverse. Kindly work on the styling of your application. It doesn't have to be flashy, but still you can improve the layout and UI. Eg., the scoreboard has a border around it in the wireframe, you can consider adding that. Give some margin between the input taking part. Move the top heading to the left side.
| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/216621525-bab3daa7-b67c-4e75-ac00-160f71284cc0.png)| ![image](https://user-images.githubusercontent.com/33268927/216621650-1cb238c6-329d-4206-b88f-d628fc19a343.png)|